### PR TITLE
fix: typeface broken slider

### LINKED ISSUE #544: [IDL] Typeface page > Slider is broken
Slider component no worky

https://www.ibm.com/design/language/typography/typeface#type-tester

### PR DESCRIPTION
**Pull request by jnm2377**
Closes #544

### CODE REVIEW

File: src/components/TypeTester/TypeTester.js [hunk 1]
```diff
@@ -338,9 +338,8 @@ export default class TypeTester extends Component {
               min={100}
               max={1600}
               value={this.state.typeSizeMultiplier}
-              key={this.state.typeSizeMultiplier}
               onChange={e => {
-                this.setState({ typeSizeMultiplier: Number(e.value) });
+                this.setState({ typeSizeMultiplier: e.value });
```
Comment (jnm2377) [inline src/components/TypeTester/TypeTester.js:342]: for reviewers: `e.value` is already a number

File: src/components/TypeTester/TypeTester.js [hunk 2]
```diff
@@ -338,9 +338,8 @@ export default class TypeTester extends Component {
               min={100}
               max={1600}
               value={this.state.typeSizeMultiplier}
-              key={this.state.typeSizeMultiplier}
```
Comment (jnm2377) [inline src/components/TypeTester/TypeTester.js:341]: @vpicone don't know why, but this line of code is what was breaking the slider. It did not like having a key 👀

**Comment by andreancardona** [comment]
@jnm2377 LGTM :)

File: src/components/TypeTester/TypeTester.js (hunk 2 above)
Comment (vpicone) [reply src/components/TypeTester/TypeTester.js:341]: @jnm2377 that was a hacky work around against an upstream Carbon issue: https://github.com/carbon-design-system/carbon/issues/6039

Without it, nothing happened at all. Now that we've updated Carbon removing this was the right move!

**Pull request merged**